docs/userguide.md

```diff
@@ -100,7 +100,7 @@ on your Mac by clicking on the folders in the "Edit Files" section of the
 container summary screen.
 
 This allows you to manage files in volumes via the Finder.
-Kitematic exposes a container's volume data under `~/Kitematic/<container's name>/`.
+Kitematic exposes a container's volume data under `~/Kitematic/Documents/<container's name>/`.
```

Comment:
I believe this to be incorrect - The new location should be:
`~/Documents/Kitematic/<container's name>/`